ailab/README.md

```diff
@@ -0,0 +1,34 @@
+# Launch Benchmark Remotely
+
+You can set up a server and and launch the benchmark run remotely.
+
+## Server Setup
+
+#### Install DJango
+```
+pip install Django
+```
+
+#### Config MySQL Database
+- Install MySQL on the server
```

Comment:
can you provide examples on how to install mysql on the server, and how to create the database?